config/core/test-configs.yaml

```diff
@@ -2332,6 +2356,14 @@ test_configs:
     test_plans:
       - baseline_qemu
 
+  - device_type: r8a7743-iwg20d-q7
+    test_plans:
+      - baseline
```

Comment:
Failing in LAVA: https://lava.ciplatform.org/scheduler/job/480726

Reply:
See other comment above, it's still failing to boot.

Reply:
see replay above